src/test/test_unite_block_fixture.cpp

```diff
@@ -40,8 +40,10 @@ CBlock GetRealBlock() {
 
           // tx[0] coinbase transaction
 
+          // tx type
+          "00"
           // tx version
-          "01000000"
+          "01"
```

Comment:
I see we had a version and then type before and now it's opposite. I think it makes more sense to keep the previous order as it makes it easier to compare the change.